presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigUtils.java

```diff
@@ -74,4 +74,123 @@ public static void deployConfigFiles(File configTar, Path configDir)
             throw new PrestoManagerException("Failed to deploy config files", untarResult);
         }
     }
+
+    public static void addConfigFiles(Path configDir)
+            throws PrestoManagerException
+    {
+        try {
+            deleteDirectoryContents(configDir);
+            if (containsRegularFiles(DEFAULT_CONFIG_DIR)) {
```

Comment:
I think this is not a good idea., if the Presto config files change in the future.